src/go/k8s/config/crd/kustomization.yaml

```diff
@@ -1,29 +1,47 @@
 # This kustomization.yaml is not intended to be run by itself,
 # since it depends on service name and namespace that are out of this kustomize package.
 # It should be run by config/default
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+#+kubebuilder:scaffold:crdkustomizeresource
```

Comment:
NIT: The same comment as with patches. Kubebuilder can be confused if we would like to add new resource later via kubebuilder cli.

Reply:
Oh, I know what happened. I did a `kustomize edit fix` and it reorganized. I'll open a PR to move these back where they belong.